tests/parser/test_parser.py

```diff
@@ -0,0 +1,11 @@
+from pathlib import Path
+
+from src.pytest_bdd.gherkin_parser import get_gherkin_document
+
+
+def test_parser():
+    test_dir = Path(__file__).parent
+    feature_file = test_dir / "test.feature"
+    feature_file_path = str(feature_file.resolve())
+
+    get_gherkin_document(feature_file_path)
```

Comment:
it'd be great if we can assert the whole structure of this gherkin document, to make sure it is parsed correctly

Reply:
Added to test assertion